pinot-core/src/main/java/org/apache/pinot/core/operator/filter/predicate/RangePredicateEvaluatorFactory.java

```diff
@@ -129,6 +129,14 @@ public static BaseRawValueBasedPredicateEvaluator newRawValueBasedEvaluator(Rang
       }
     }
 
+    public int getStartDictId() {
+      return _startDictId;
+    }
+
+    public int getEndDictId() {
+      return _endDictId - 1;
```

Comment:
Let's return `_endDictId` and do this `-1` at the caller side with comments for readability

Reply:
Done.